### PR TITLE
Close OO connection on JabRef exit

### LINKED ISSUE #9075: App hang on exit on macOS
### JabRef version

Latest development branch build (please note build date below)

### Operating system

macOS

### Details on version and operating system

JabRef 5.8--2022-08-16--6a71395 Mac OS X 12.5.1 x86_64  Java 18.0.2  JavaFX 18.0.2+2

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Run JabRef, use LibreOffice panel to connect to a document and run.
2. Try to quit JabRef.
4. Observe that JabRef window closes but the app doesn't exit; needs force quit.

### Appendix

Log file (hang log from Apple crash reporter):

[Hang.log.zip](https://github.com/JabRef/jabref/files/9388110/Hang.log.zip)

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fixes #9075 

Closes the OO-Bridge when JabRef exits. If none is open, nothing happens. Otherwise the bridge is closed, JabRef exits but the LO document stays open

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Review by calixtus — APPROVED** [review]
LGTM

File: src/main/java/org/jabref/gui/openoffice/OOBibBaseConnect.java [hunk 1]
```diff
@@ -70,6 +78,27 @@ private XDesktop simpleBootstrap(Path loPath)
...
+                    .createInstance("com.sun.star.bridge.BridgeFactory"));
+
+            if (bridgeFactory != null) {
+                for (XBridge bridge : bridgeFactory.getExistingBridges()) {
+                    // dispose of this bridge after closing its connection
+                    queryInterface(XComponent.class, bridge).dispose();
+                }
+            }
+        } catch (Exception ex) {
+            LOGGER.error("Exception disposing office process connection bridge:", ex);
```
Comment (koppor) [inline src/main/java/org/jabref/gui/openoffice/OOBibBaseConnect.java:98]:
```suggestion
            LOGGER.error("Exception disposing office process connection bridge", ex);
```

File: src/main/java/org/jabref/gui/openoffice/OOBibBaseConnect.java [hunk 2]
```diff
@@ -70,6 +78,27 @@ private XDesktop simpleBootstrap(Path loPath)
         return UnoCast.cast(XDesktop.class, desktop).get();
     }
 
+    /**
+     *  Close any open office connection, if none exists does nothing
```
Comment (koppor) [inline src/main/java/org/jabref/gui/openoffice/OOBibBaseConnect.java:82]:
```suggestion
     * Close any open office connection, if none exists does nothing
```

**Review by koppor — COMMENTED** [review]
Two nitpicks. Then LGTM.

**Pull request merged**